scripts/get_redash_api_key.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+# Check if the namespace was provided as an argument
+if [ -z "$1" ]; then
+  echo "Namespace not provided. Exiting..."
+  exit 1
+fi
+
+# Retrieve the namespace from the argument
+namespace="$1"
+
+# Prompt the user for their Redash username
+read -p "Enter your Redash username: " USERNAME
+
+# Prompt the user for their Redash password (input hidden for security)
+read -s -p "Enter your Redash password: " PASSWORD
+echo
+
+REDASH_URL="https://forms-flow-analytics-$namespace.aot-technologies.com"

```

Comment:
Can you make the REDASH_URL also as a variable than hardcoding with our domain and ingress ?

Reply:
ok.. will make the changes